src/components/Layout/LocalizationWidget/LocalizationWidget.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useReducer } from "react";
+import { strings } from "../../../assets/LocalizedStrings";
+import { Dropdown, Button } from "semantic-ui-react";
+import { SUPPORTED_LANGUAGES } from "../../../constants/ProjectConstants";
+
+function DropdownExampleSearchDropdown() {
+  const [ignored, forceUpdate] = useReducer((x) => x + 1, 0);
+
+  const languageOptions = SUPPORTED_LANGUAGES.map((supportedLanguage) => {
+    return {
+      key: supportedLanguage,
+      text: supportedLanguage,
```

Comment:
Agree but even more so:

"English (en), Espanol (es), Deutsch (de)"

Reply:
I ultimately went with the full language name in the locale that it would make sense in.  So for example, if you speak _only_ Spanish and you click on the localization button, you'll see "Español" even though the "current" language is English.  That is the way that folks like Apple do it, and it seems like good practice particularly for character-based languages readers might be confused having to pick their own language out of a list of 'alphabet' languages.